modules/op-deleting-the-pipelines-component-and-custom-resources.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// */openshift_pipelines/uninstalling-pipelines.adoc
+
+[id='op-deleting-the-pipelines-component-and-custom-resources_{context}']
+= Deleting the {pipelines-title} components and custom resources
+
+Delete the custom resources created by default during installation of the {pipelines-title} Operator.
+
+[discrete]
+.Procedure
+. In the *Administrator* perspective of the Web Console, navigate to *Administration->Custom Resource Definition*.
+
+. Type `config.operator.tekton.dev` in *Filter by name* box to search for the {pipelines-title} Operator custom resources.
+
+. Click on the *CRD Config* to see *Custom Resource Definition Details* page.
+
+. Click on the *Action* drop-down menu and Select *Delete Custom Resource Definition*.
```

Comment:
Click **CRD Config** to..
Click the *Action* drop-down menu...
Do not use preposition 'on'

Reply:
done